web/js/map-OpenLayers.js

```diff
@@ -144,6 +173,25 @@ function fixmystreet_onload() {
             backgroundHeight: 40,
             backgroundXOffset: -10,
             backgroundYOffset: -35
+        },
+        'small': {
+            externalGraphic: fixmystreet.pin_prefix + "pin-${colour}-small.png",
+            graphicWidth: 24,
+            graphicHeight: 32,
+            graphicXOffset: -12,
+            graphicYOffset: -32,
```

Comment:
Good catch; thanks. I think the offset numbers here are to do with positioning of the marker image relative the the point it's marking. It looks as though the bubble offset is set [elsewhere](https://github.com/mysociety/fixmystreet/blob/master/web/js/map-OpenLayers.js#L193) and will need to vary depending on the marker size in use.